automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspaceApiSpec.scala

```diff
@@ -62,6 +64,46 @@ class WorkspaceApiSpec extends TestKit(ActorSystem("MySpec")) with AnyFreeSpecLi
         }
       }
 
+      "to create, clone, and delete v2 workspaces (using the Resource Buffer), in a v2 billing project" in {
+        val owner: Credentials = UserPool.chooseProjectOwner
+        implicit val ownerAuthToken: AuthToken = owner.makeAuthToken(AuthTokenScopes.billingScopes)
+        val billingProjectName = uuidWithPrefix("WorkspaceApiSpec_createCloneDeleteUsingResourceBuffer")
+        Rawls.billingV2.createBillingProject(billingProjectName, ServiceTestConfig.Projects.billingAccountId)
+        val workspaceName = prependUUID("rbs-test-workspace")
+        val workspaceCloneName = s"$workspaceName-clone"
+
+        implicit val ec: ExecutionContext = ExecutionContext.global
+        val source = scala.io.Source.fromFile(RawlsConfig.pathToQAJson)
+        val jsonCreds = try source.mkString finally source.close() // read in the json file and safely close it
```

Comment:
I think the code here successfully says what it means, I don't think you need the comment.